src/dataflow/src/source/kafka.rs

```diff
@@ -135,11 +117,6 @@ impl SourceInfo<Vec<u8>> for KafkaSourceInfo {
             }
         }
         self.known_partitions = cmp::max(self.known_partitions, pid + 1);
-
-        assert_eq!(
-            self.get_worker_partition_count(),
-            self.get_partition_consumers_count()
-        );
```

Comment:
Can `get_worker_partition_count` be moved to KafkaSourceInfo so we can keep the invariant assertions for the kafka source? Or do you think these have no value?

Reply:
i felt like these had no value